lib/hal/video.c

```diff
@@ -428,12 +428,12 @@ void XVideoSetVideoEnable(BOOL enable)
 
 void XVideoSetFlickerFilter(int level)
 {
-	if(!flickerSet || (level & 0x07) != flickerLevel)
+	if(!flickerSet || (level & 0x05) != flickerLevel)
```

Comment:
If this mask is intended to clamp it to be <=5, this isn't right?
This will mask out 2 and 3 aswell.

What's the intent of this?

Reply:
👍reverted, any non-valid value will set the max value.